themis/_index.md

```diff
@@ -40,7 +82,22 @@ We created Themis to build other products on top of it - i.e. [Acra](https://www
 
 ## What Themis doesn’t solve
 
-TODO:
+
+1. Themis doesn't provide a low-level/raw cryptographic API for encryption and hashing.
+
+   For example, Themis doesn't provide a way to select a certain cryptographic cipher, or key length, or exact elliptic curve, or hash function (there is no API like `encrypt(cipher: AES, keylenght:256, mode:GCM)`). 
+
+   Instead of thinking about cryptographic parameters, developers take Themis to solve certain functions/requirement of their products. Themis is designed in a way that eliminates typical cryptographic mistakes, and makes development faster, but Themis is not as flexible as OpenSSL.
+
+   For those who understand what they are doing and require certain ciphers, [Themis allows to change default algorithms and build a custom version](/themis/installation/installation-from-sources/#selecting-algorithm-parameters).
+
+2. Themis doesn't have an API for homomorphic or PQ encryption.
+
+3. Themis is not designed to run on low-energy hardware, 16-bit microcontrollers and big-endian devices. Themis supports 32-bits/64-bits CPUs, little-endian.
```

Comment:
```suggestion
3. Themis is not designed to run on low-power hardware, microcontrollers, etc.
   Themis targets mobile, desktop, and server hardware with x86 and ARM CPUs.
```

I believe that lack of support for 16-bit architectures and big-endian systems is a programming flaw, not a design decision. Though, by using OpenSSL we focus on commodity hardware and non-embedded use cases.